scripts/dev.sh

```
@@ -0,0 +1,242 @@
+            fi
+        else
+            info "Skipping setup. You can run './scripts/quick_setup.sh' anytime."
+            # Create marker to not ask again
+            touch "$SETUP_MARKER"
+            echo ""
+        fi
+    fi
+fi
+
+# Set environment variables for faster builds
+export CARGO_BUILD_JOBS=$(nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || echo 4)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)